src/components/ResumePageContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import { Button } from '@/components/ui/button';
+
+export function ResumePageContainer() {
+  return (
+    <div className="w-4xl max-w-4xl mx-auto p-6 space-y-6">
+      {/* Embedded PDF Preview */}
+      <div className="w-full h-[600px] rounded-2xl overflow-hidden shadow-md">
+        <iframe
+          src="/crc_resume.pdf"
+          className="w-full h-full"
+          style={{ border: 'none' }}
+        />
+      </div>
+      {/* Download Button */}
+      <div className="flex justify-center">
+        <Button asChild>
+          <a href="/crc_resume.pdf" download>
+            Download Résumé (PDF)
+          </a>
+        </Button>
+      </div>
```

Comment:
Could probably get away with either iframe or a here- but they have tradeoffs. `iframe`s are shit for mobile viewing/slow connection speed; `a` element doesn't allow an actual previewable...

am thinking about some middle ground where I wrap an `img` in the `a` element to give a previewable image that also is downloadable- so it always works on mobile too... Thoughts?

Leaving as is for now, just to get things going on the resume page.